Day1/P1.py

```diff
@@ -0,0 +1,300 @@
+        elif txt[i] == 'U':
+            txt = txt[0:i] + 'u' + txt[i+1:]
+        elif txt[i] == 'V':
+            txt = txt[0:i] + 'v' + txt[i+1:]
+        elif txt[i] == 'W':
+            txt = txt[0:i] + 'w' + txt[i+1:]
+        elif txt[i] == 'X':
+            txt = txt[0:i] + 'x' + txt[i+1:]
+        elif txt[i] == 'Y':
+            txt = txt[0:i] + 'y' + txt[i+1:]
+        elif txt[i] == 'Z':
+            txt = txt[0:i] + 'z' + txt[i+1:]
+    return txt
+
+print(lowerstr('Hello, And Welcome To My World!'))
+print(lowerstr('HSKHNKXjkjskfeKWEOIIHF'))
+print(lowerstr('hello how are you'))
+print('-'*20)
+
+#upper case all characters
+def upperstr(txt):
+    for i in range(len(txt)):
+        if txt[i] == 'a':
+            txt = txt[0:i] + 'A' + txt[i+1:]
+        elif txt[i] == 'b':
+            txt = txt[0:i] + 'B' + txt[i+1:]
+        elif txt[i] == 'c':
+            txt = txt[0:i] + 'D' + txt[i+1:]
+        elif txt[i] == 'd':
+            txt = txt[0:i] + 'D' + txt[i+1:]
+        elif txt[i] == 'e':
+            txt = txt[0:i] + 'E' + txt[i+1:]
+        elif txt[i] == 'f':
+            txt = txt[0:i] + 'F' + txt[i+1:]
+        elif txt[i] == 'g':
+            txt = txt[0:i] + 'G' + txt[i+1:]
+        elif txt[i] == 'h':
+            txt = txt[0:i] + 'H' + txt[i+1:]
+        elif txt[i] == 'i':
+            txt = txt[0:i] + 'I' + txt[i+1:]
+        elif txt[i] == 'j':
+            txt = txt[0:i] + 'J' + txt[i+1:]
+        elif txt[i] == 'k':
+            txt = txt[0:i] + 'K' + txt[i+1:]
+        elif txt[i] == 'l':
+            txt = txt[0:i] + 'L' + txt[i+1:]
+        elif txt[i] == 'm':
+            txt = txt[0:i] + 'M' + txt[i+1:]
+        elif txt[i] == 'n':
+            txt = txt[0:i] + 'N' + txt[i+1:]
+        elif txt[i] == 'o':
+            txt = txt[0:i] + 'O' + txt[i+1:]
+        elif txt[i] == 'p':
+            txt = txt[0:i] + 'P' + txt[i+1:]
+        elif txt[i] == 'q':
+            txt = txt[0:i] + 'Q' + txt[i+1:]
+        elif txt[i] == 'r':
+            txt = txt[0:i] + 'R' + txt[i+1:]
+        elif txt[i] == 's':
+            txt = txt[0:i] + 'S' + txt[i+1:]
+        elif txt[i] == 't':
+            txt = txt[0:i] + 'T' + txt[i+1:]
+        elif txt[i] == 'u':
+            txt = txt[0:i] + 'U' + txt[i+1:]
+        elif txt[i] == 'v':
+            txt = txt[0:i] + 'V' + txt[i+1:]
+        elif txt[i] == 'w':
+            txt = txt[0:i] + 'W' + txt[i+1:]
+        elif txt[i] == 'x':
+            txt = txt[0:i] + 'X' + txt[i+1:]
+        elif txt[i] == 'y':
+            txt = txt[0:i] + 'Y' + txt[i+1:]
+        elif txt[i] == 'z':
+            txt = txt[0:i] + 'Z' + txt[i+1:]
+    return txt
+
+print(upperstr('Hello, And Welcome To My World!'))
+print(upperstr('HSKHNKXjkjskfeKWEOIIHF'))
+print(upperstr('HELLO HOW ARE YOU'))
+print('-'*20)
+
+#count the number of times a specified value appears in the string
+def countvalue(txt,char,start=None,end=None):
+    count = 0
+    for i in range(len(findchars(txt,char,start,end))):
```

Comment:
State your idea in English writing first. Then translate this for loop into English and try to understand what it is doing here? Is it what you want with your initial idea? What is the value of count? What are you trying to do with `len(....)` and `count`?

Reply:
As when function findchars is called, the value returned is the list of the positions the characters appear in a string. In order to count the number of times a specified value appears in the string, I count the number of elements in the list using for loop to go through the list. In this case I will remove `range(len(...))` and instead call for loop `for i in findchars()` so that each time an element is looped over, count would increase by 1.